noodles-editor/src/render/renderer.ts

```diff
@@ -273,19 +273,10 @@ export const useRenderer = ({
     return () => cancelAnimationFrame(tick)
   }, [isRendering])
 
-  // Used to trigger a re-render of the canvas in Deck when the render is stuck
-  const [_animate, setAnimate] = useState(false)
-  const advanceFrame = useCallback(() => {
-    setAnimate(true)
-    requestAnimationFrame(() => setAnimate(false))
-  }, [])
-
   return {
     startCapture,
     captureFrame,
     currentFrame: currentFrame.current,
-    advanceFrame,
```

Comment:
This old advance frame action hasn't helped "stuck" renders in noodles and was left over from the old visTypes frameworks. Frames that don't cause any deck props to update can still get stuck, but this action didn't help. The root cause has to do with how we do frame signaling